### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Sites-Edit-Web Application_Color Contrast: Color Contrast ratio fails for "Web server > Web Sites" link text.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#166]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106

**MAS ID Violated:** MAS1.4.3

**Repro Steps:**
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till "Web Sites" in the left menu section and Press Enter.
5. Tab till "More" commands button of the existing website inside the grid and Press Enter.
6. Tab till "Edit" in the expanded menu and Press Enter.
7. Tab till "Web Applications" in the left menu section and Press Enter.
8. Tab till "Create Web Application" button and Press Enter.
9. Navigate to Application Pool page tab.
10. Verify the color contrast ratio is at least 4.5:1 for all the text on the page.

**Actual Result:**
Color Contrast ratio (4.0:1) fails for "Web server > Web Sites" link text.

**Expected Result:**
Minimum color contrast ratio for the normal text should be greater than or equal to 4.5:1.

**Note**: Same issue is observed on "Virtual Directories" page as well.

**User Impact:**
People with low vision often have difficulty reading text that does not contrast with its background. This can be exacerbated if the person has a color vision deficiency that lowers the contrast even further. Providing a minimum luminance contrast ratio between the text and its background can make the text more readable even if the person does not see the full range of colors. It also works for the rare individuals who see no color.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Attachment:**
![bug 166_color contrast issue](https://user-images.githubusercontent.com/38521133/44779601-8b4b8e80-ab9d-11e8-9c1f-5ab3b6ffda41.png)

**Comment by jhkimnew** [comment]
Fixed removed color attribute

**Comment by mstechie** [comment]
GitHubTags:#A11y_Manage.IIS.Net;
